test/algolia_test.exs

```diff
@@ -4,19 +4,26 @@ defmodule AlgoliaTest do
   import Algolia
 
   @indexes [
-    "test", "test_1", "test_2", "test_3", "test_4",
```

Comment:
I'd rather make a separate PR to `mix format` the whole repo instead, to make the review easier.